src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml

```diff
@@ -65,24 +63,22 @@ tests:
 
     ### Optional Depends on the FeatureMap
     - label:
-          "1d: If (PA & LF) TH reads CurrentPositionLiftPercentage optional
-          attribute from DUT"
+          "1d: TH reads CurrentPositionLiftPercentage optional attribute from
+          DUT"
       command: "readAttribute"
       attribute: "CurrentPositionLiftPercentage"
-      PICS: WNCV.S.F00 && WNCV.S.F02 && WNCV.S.A0008
+      PICS: WNCV.S.A0008 && ( WNCV.S.F00 || WNCV.S.F01 )
```

Comment:
wrong

Reply:
Removed changes from PR please verify it.